config/src/lib.rs

```diff
@@ -1079,11 +1092,7 @@ impl Config {
         if !chain_path.exists() {
             return Ok(blocks)
         }
-        for block in chain_path
-            .read_dir()?
-            .flatten()
-            .filter(|x| x.file_type().unwrap().is_dir() && !x.file_name().eq("etherscan"))
```

Comment:
removed the "etherscan" filter as I changed the cache directory structure to hoist the etherscan cache outside of this directory.